KSTokenView/KSToken.swift

```diff
@@ -174,8 +174,5 @@ class KSToken : UIControl {
          rectanglePath.stroke()
       }
    }
-   
-   func description() -> String {
```

Comment:
I see.  What we could do is `override var description: String { return title }` to keep with the authors intentions.  I don't really care either way